2.feature_extraction/0.create_loaddata_csvs.ipynb

```diff
@@ -405,11 +405,11 @@
        "3     r03c03f04p01-ch1sk1fk1fl1.tiff   \n",
        "4     r03c03f05p01-ch1sk1fk1fl1.tiff   \n",
        "...                              ...   \n",
-       "2842  r14c12f05p01-ch1sk1fk1fl1.tiff   \n",
-       "2843  r14c12f06p01-ch1sk1fk1fl1.tiff   \n",
-       "2844  r14c12f07p01-ch1sk1fk1fl1.tiff   \n",
-       "2845  r14c12f08p01-ch1sk1fk1fl1.tiff   \n",
-       "2846  r14c12f09p01-ch1sk1fk1fl1.tiff   \n",
+       "5002  r14c12f05p01-ch1sk1fk1fl1.tiff   \n",
+       "5003  r14c12f06p01-ch1sk1fk1fl1.tiff   \n",
+       "5004  r14c12f07p01-ch1sk1fk1fl1.tiff   \n",
+       "5005  r14c12f08p01-ch1sk1fk1fl1.tiff   \n",
+       "5006  r14c12f09p01-ch1sk1fk1fl1.tiff   \n",
```

Comment:
I approved this the first time around, but I’m doing a second run-through. I noticed that these values are different. Based on the changes you conducted, is this an expected outcome?

Reply:
Huh I mean the loaddata CSVs never changed so I am not sure why it is showing as different. Looks the same to me lol